x-pack/platform/packages/shared/kbn-sample-parser/client/create_loghub_generator.ts

```diff
@@ -73,6 +73,7 @@ export function createLoghubGenerator({
         const next = {
           '@timestamp': simulatedTimestamp,
           message: parser.replaceTimestamp(line.message, simulatedTimestamp),
+          ...parser.getFakeMetadata(line.message),
```

Comment:
should this be opt-in or opt-out?

Reply:
it has some implications on e.g. sig events but probably fine!

Reply:
What are the implications?

Reply:
Mostly around testing, eg this will skew the numbers because the payload size for fetching and bulk indexing gets bigger. But its totally fine